src/App.module.css

```diff
@@ -2,8 +2,14 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
 }
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.), Please add the correct font families for your app,` Gill sans` and `Lato` fonts, not `Montserrat` as you did.